Missing install command, Typo in Typescript code

Copied from https://github.com/MicrosoftDocs/feedback/issues/968

**Link to article:**
https://docs.microsoft.com/en-us/graph/tutorials/angular?tutorial-step=1

**Problem:**

**Missing**
(ERROR in src/app/app.module.ts(1,27): error TS2307: Cannot find module '@ng-bootstrap/ng-bootstrap'.):

```
npm install --save @ng-bootstrap/ng-bootstrap
```

**Typo**

(AppRoutingModule is written twice):

```
imports: [ BrowserModule, AppRoutingModule AppRoutingModule, NgbModule, FontAwesomeModule ]
```

Should be:

```
imports: [ BrowserModule, AppRoutingModule, NgbModule, FontAwesomeModule ]
```

**Typo**

(In both nav-bar.component.ts and home.component.ts, tslint wants ' rather than "):

```
signIn(): void { 
  // Temporary this.authenticated = true; 
  this.user = { displayName: "Adele Vance", email: "AdeleV@contoso.com" }; 
}
```

Should be:

```
signIn(): void { 
  // Temporary this.authenticated = true; 
  this.user = { displayName: 'Adele Vance', email: 'AdeleV@contoso.com' }; 
}
```

--- a/demos/01-create-app/graph-tutorial/src/app/home/home.component.ts
+++ b/demos/01-create-app/graph-tutorial/src/app/home/home.component.ts
@@ -23,8 +23,8 @@ export class HomeComponent implements OnInit {
     // Temporary
     this.authenticated = true;
     this.user = {
-      displayName: "Adele Vance",
-      email: "adelev@contoso.com"
+      displayName: 'Adele Vance',
+      email: 'adelev@contoso.com'
     };
   }
 }

--- a/demos/01-create-app/graph-tutorial/src/app/nav-bar/nav-bar.component.ts
+++ b/demos/01-create-app/graph-tutorial/src/app/nav-bar/nav-bar.component.ts
@@ -32,8 +32,8 @@ export class NavBarComponent implements OnInit {
     // Temporary
     this.authenticated = true;
     this.user = {
-      displayName: "Adele Vance",
-      email: "adelev@contoso.com"
+      displayName: 'Adele Vance',
+      email: 'adelev@contoso.com'
     };
   }
 

--- a/tutorial/create-app.md
+++ b/tutorial/create-app.md
@@ -38,7 +38,7 @@ Run the following command in your CLI.
 ```Shell
 npm install bootstrap@4.1.3 @fortawesome/angular-fontawesome@0.3.0 @fortawesome/fontawesome-svg-core@1.2.8
 npm install @fortawesome/free-regular-svg-icons@5.5.0 @fortawesome/free-solid-svg-icons@5.5.0
-npm install moment@2.22.2 moment-timezone@0.5.23
+npm install moment@2.22.2 moment-timezone@0.5.23 @ng-bootstrap/ng-bootstrap@4.0.0
 npm install @azure/msal-angular@0.1.2 rxjs-compat@6.3.3 @microsoft/microsoft-graph-client@1.3.0
 ```
 
@@ -84,7 +84,6 @@ In the `@NgModule` declaration, replace the existing `imports` array with the fo
 ```TypeScript
 imports: [
   BrowserModule,
-  AppRoutingModule
   AppRoutingModule,
   NgbModule,
   FontAwesomeModule
@@ -134,8 +133,8 @@ export class NavBarComponent implements OnInit {
     // Temporary
     this.authenticated = true;
     this.user = {
-      displayName: "Adele Vance",
-      email: "AdeleV@contoso.com"
+      displayName: 'Adele Vance',
+      email: 'AdeleV@contoso.com'
     };
   }
 
@@ -234,8 +233,8 @@ export class HomeComponent implements OnInit {
     // Temporary
     this.authenticated = true;
     this.user = {
-      displayName: "Adele Vance",
-      email: "AdeleV@contoso.com"
+      displayName: 'Adele Vance',
+      email: 'AdeleV@contoso.com'
     };
   }
 }

